plugin-annotation/src/test/java/com/mapbox/maps/plugin/annotation/generated/PolylineAnnotationManagerTest.kt

```diff
@@ -634,4 +638,4 @@ class PolylineAnnotationManagerTest {
     verify(exactly = 1) { manager.layer?.lineWidth(Expression.get(PolylineAnnotationOptions.PROPERTY_LINE_WIDTH)) }
     verify(exactly = 1) { manager.dragLayer?.lineWidth(Expression.get(PolylineAnnotationOptions.PROPERTY_LINE_WIDTH)) }
   }
-}
\ No newline at end of file
+}
```

Comment:
nit: I guess ktlint will complain the new line at the end of the source file 🙂

Reply:
Yup